content/post/52-go-ipfs-0.4.18.md

```diff
@@ -0,0 +1,283 @@
+
+# Voila - point your browser at http://localhost:5551/webui to inspect your remote
+```
+
+#### `ipfs name resolve` streaming response
+
+There is now a new flag for `ipfs name resolve` - `--stream`. When the command
+is invoked with the flag set, it will start returning results as soon as they
+are discovered in the DHT and other routing mechanisms. This enables certain
+applications to start prefetching/displaying data while the discovery is still
+running. Note that this command will likely return many outdated records
+before it finding and returning the latest. However, it will always return
+*valid* records (even if a bit stale).
+
+#### `ipfs add` block inlining
+
+In the previous release, we added support for extracting blocks inlined
+into CIDs. In this release, we've added support for creating these CIDs. You can
+now run `ipfs add` with the `--inline` flag to inline blocks less than or equal
+to 32 bytes in length into a CID, instead of writing an actual block. This
+should significantly reduce the size of filesystem trees with many empty
+directories and tiny files.
+
+## WebUI
+
+This release includes the latest, very shiny [updated webui](https://github.com/ipfs-shipyard/ipfs-webui). You can view it by
+installing go-ipfs and visiting http://localhost:5001/webui. It deserves its own release note - oh look, it [got
+one](./51-js-ipfs-0-33/#web-ui-2-0)! Here's a peek:
+
+![Screenshot of the status page](https://raw.githubusercontent.com/ipfs-shipyard/ipfs-webui/master/docs/screenshots/ipfs-webui-status.png)
+
+| Files | Explore | Peers | Settings |
+|-------|---------|-------|----------|
+| ![Screenshot of the file browser page](https://raw.githubusercontent.com/ipfs-shipyard/ipfs-webui/master/docs/screenshots/ipfs-webui-files.png) | ![Screenshot of the IPLD explorer page](https://raw.githubusercontent.com/ipfs-shipyard/ipfs-webui/master/docs/screenshots/ipfs-webui-explore.png) | ![Screenshot of the swarm peers map](https://raw.githubusercontent.com/ipfs-shipyard/ipfs-webui/master/docs/screenshots/ipfs-webui-peers.png) | ![Screenshot of the settings page](https://raw.githubusercontent.com/ipfs-shipyard/ipfs-webui/master/docs/screenshots/ipfs-webui-settings.png) |
+
+
+Kudos and thanks to the webui team!
+
+## Performance
+
+This release includes some significant performance improvements, both in terms
+of resource utilization and speed. This section will go into some technical
+details so feel free to skip it if you're just looking for shiny new features.
+
+### Resource Utilization
+
+In this release, we've (a) fixed a slow memory leak in libp2p and (b)
+significantly reduced the allocation load. Together, these should improve both
+memory and CPU usage.
+
+#### Datastructures
+
+We've changed two of our most frequently used datastructures, CIDs and
+Multiaddrs, to reduce allocation load.
+
+First, we now store CIDs *encode* as strings, instead of decoded in structs
```

Comment:
This doesn't read properly- should it be "encoded", "CID's _encode_ value", or something else?

Reply:
Yah - encoded I think.